src/Config.cpp

```diff
@@ -252,6 +252,7 @@ void Config::Load(const std::string& overrideConfigLocation)
         AddConfigVar("WidgetsRight", config.widgetsRight, lineView, foundProperty);
 
         AddConfigVar("CPUThermalZone", config.cpuThermalZone, lineView, foundProperty);
+        AddConfigVar("GPUThermalZone", config.gpuThermalZone, lineView, foundProperty);
```

Comment:
The name should incorporate that this is only for AMD GPUs, so there is no confusion whether you actually need to set it. I'd suggest "AmdGPUThermalZone"